### DISCUSSION TITLE
access problems with all dapps

### DISCUSSION THREAD

**Discussion by manuelmoli** [WalletConnect/walletconnect-monorepo#4216] [Developer Support]
Hello, I have not been able to access any dapps through walletconnect for several days using a hardware wallet (keepkey).
 On the last screen of walletconnect, the account is left pending, and in the end the connection is rejected (see attached images).
 Any solution?
 Thanks in advance.
![Screenshot_20240207-164006](https://github.com/WalletConnect/walletconnect-monorepo/assets/159289186/864cd9a1-a978-48e7-8ffa-a010386da8e9)
![Screenshot_20240207-163733~2](https://github.com/WalletConnect/walletconnect-monorepo/assets/159289186/6cd4ce16-00a9-4122-874c-03fe9cc209d9)

**Comment by glitch-txs** [comment]
please contact Keepkey support team 🙏
